docs/docs/upgrading-node-js.md

```diff
@@ -0,0 +1,119 @@
+
+## What version of Node do I have?
+
+Run `node -v` in a terminal to see which version of Node you have.
+
+```
+node -v
+v10.16.0
+```
+
+This example shows Node version 10.
+
+## Upgrading from Node version 6
+
+Node version 6 reached _End-of-life_ status on 30th April 2019. Many of Gatsby's dependencies are updating to Node version 8 and above. Gatsby must also update in order release new features and bug fixes.
+
+Generally we'd recommend using [the Node version who's status is _Active LTS_](https://github.com/nodejs/Release#nodejs-release-working-group) (Node 10 at time of writing). However, in this document you'll learn how to update from Node 6 to Node 8 as this is likely to be the least disruptive upgrade for you.
+
+> What about Node 7? Stable versions of Node are evenly numbered releases - Node 6, Node 8, Node 10 etc. Only use uneven release numbers if you'd like to try cutting-edge and experimental features.
+
+There are multiple ways to update your version of Node depending on how you originally installed it. Read on to find the best approach for you.
+
+### Using Homebrew
+
+This is our recommended way to install a newer version of Node.
+
+You will have homebrew installed on your computer if you [followed part zero of the Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-zero/#-install-nodejs-and-npm). Homebrew is a program that allows you to install specific versions of Node (and other software).
+
+To update from Node 6 to Node 8 using Homebrew, open a terminal and run the following commands:
+
+```
+brew search node
+```
+
+You should see output similar to this:
+
+```
+brew search node
+==> Formulae
+heroku/brew/heroku-node ✔        llnode                           node@10                          nodebrew
+leafnode                         node ✔                           node@8                           nodeenv
+libbitcoin-node                  node-build                       node_exporter                    nodenv
+```
+
+You're interested in the next stable version of Node after Node 6, which is Node 8. Homebrew makes this available in a package called `node@8`. Run:
+
+```
+brew install node@8
+```
+
+Once that's complete, run:
+
+```
+node -v
+```
+
+to confirm that you've upgraded from Node version 6 up to version 8.
+
+### Using a Node version management package
+
+There are two popular packages used for managing multiple versions of Node on your system. Use one of these to update to a newer version of Node if they're already available on your computer.
+
+These packages are very useful for people that regularly work with different versions of Node.
+
+#### n
+
+Run:
+
+```
+n
+```
+
+in a terminal to see if n is installed on your system. If it's installed, you can run `n 8` to install and use Node version 8.
+
+[Check n's documentation for further instructions](https://github.com/tj/n).
+
+#### nvm
```

Comment:
Nit: but I'd maybe move nvm above n? Perhaps it's just my bubble--but it seems that nvm is much more heavily used.

Reply:
Makes sense, will update 👍 

I use `nvm` but have to refer to the docs _every_ time I use it. `n`'s API is much simpler in comparison.